python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -126,10 +127,7 @@ def reload(self):
         self.registry._lock.acquire()
         try:
 
-            if self.registry.checkShouldFlush():
```

Comment:
Just to check - I'm guessing that as it is an `IncompleteObject` this will always return `False` and that's why it is being removed?